src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    Random random = new Random();
+    int randomInt = random.nextInt(100);
```

Comment:
The `randomInt` and `randomColor` are initialized only once and reused for every `Ball` object. This means all `Ball` objects will have the same color and number. Move the random number generation and color assignment inside the `getRandomBall()` method to ensure each `Ball` has a unique color and number.